README.md

```diff
@@ -211,6 +212,16 @@ oauth2Client.setCredentials({
 
 Once the client has a refresh token, access tokens will be acquired and refreshed automatically in the next call to the API.
 
+Refresh tokens may stop working after they are granted, either because:
+- The user has revoked your app's access
+- The refresh token has not been used for 6 months
+- The user changed passwords and the refresh token contains Gmail scopes
+- The user account has exceeded a max number of live refresh tokens
+- The refresh token has expired after 7 days
```

Comment:
This one doesn't make sense to me. Why would it expire after 7 days? It also seems in conflict with the reason "refresh token hasn't been used in 6 months"

Reply:
You're right, I needed to specify that it's for testing apps! I took these conditions from here: https://developers.google.com/identity/protocols/oauth2